src/contexts/CalculatorsContext.tsx

```
@@ -0,0 +1,580 @@
+        boilTimer: {
+          ...initialState.boilTimer,
+        },
+      };
+    case "SET_PREFERENCES":
+      return {
+        ...state,
+        preferences: { ...state.preferences, ...action.payload },
+      };
+    case "ADD_TO_HISTORY":
+      if (!state.preferences.saveHistory) return state;
+      const { calculatorType, inputs, result } = action.payload;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)